bzl/workspace.bzl

```diff
@@ -136,6 +143,14 @@ def plaidml_workspace():
         build_file = clean_dep("//vendor/tbb:tbb.BUILD"),
     )
 
+    http_archive(
```

Comment:
Let's keep the tensorflow stuff out of `workspace.bzl`. This file gets used by downstream repositories that may or may not want tensorflow included.